idioms/coercion-arguments.md

```diff
@@ -0,0 +1,112 @@
+# Use borrowed types for arguments
+
+## Description
+
+Using a target of a deref coercion can increase the flexibility of your code when you are deciding which argument type to use for a function argument which may increase its utility as it increases the number of viable inputs for this function.
+For example; using `&str` instead of a `&String`, or `&[T]` in preference of `&Vec<T>`, or even `&T` as opposed to a `&Box<T>`. 
```

Comment:
```suggestion
For example; using `&str` instead of `&String`, or `&Path` instead of `&PathBuf` or `&[T]` in preference of `&Vec<T>`, or even `&T` as opposed to a `&Box<T>`. 
```